packages/pages/src/dev/server/middleware/serverRenderRoute.ts

```diff
@@ -134,3 +137,23 @@ const getDocument = async (
     )
   )?.document;
 };
+
+const overrideDocument = (document: any, overrideData: any) => {
+  if (!overrideData) {
+    return document;
+  }
+  return {
+    ...document,
+    _site: {
+      ...document._site,
+      c_visualLayouts: [
```

Comment:
Mentioned in the other CR but the overrides should come through as a map of key to overridden value so that it's generic. Right now it's very specific to VE and not reusable.

Reply:
Done